examples/backbone_design/designs/l3vpn/context/__init__.py

```diff
@@ -21,9 +22,10 @@ def __hash__(self):
 
     @lru_cache
     def get_l3vpn_prefix(self, parent_prefix, prefix_length):
-        # get the next available prefix in l3vpn_prefix
-        # parent_prefix = Prefix.objects.get(prefix=parent_prefix)
-        # return parent_prefix.get_first_available_prefix()
+        existing_prefix = Prefix.objects.filter(description=self.get_instance_name("useless arg")).first()
```

Comment:
I want to find a better pattern to connect the instance_name to an existing object. Using tags would be a better option, but maybe we can find an even better one.